### PR TITLE
Add Damian Mehers as an author

### PR DESCRIPTION
**Pull request by DamianMehers**
If you are submitting a new blog please read:

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines:

- I have a valid blog & RSS URL
- Host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- Blog is active with at least 3 Xamarin related blog posts in the last 6 months
- If the blog has mixed content (Xamarin and Personal/Non-Xamarin blogs) a filter has been applied
- If you delete your blog you will come delete your blog from Planet Xamarin
- Your blog may be removed at any time if any of these are broken.

### CODE REVIEW

File: src/Firehose.Web/Authors/DamianMehers.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        public string LastName => "Mehers";
+        public string ShortBioOrTagLine => "Independent Xamarin Developer";
+        public string StateOrRegion => "Geneva, Switzerland";
+        public string EmailAddress => "damian@mehers.com";
+        public string TwitterHandle => "DamianMehers";
+        public string GravatarHash => "d77613f4e20bfcae401a6bf0018a83d1";
+        public string GitHubHandle => "DamianMehers";
+        public GeoPosition Position => new GeoPosition(46.3635288, 6.1860801);
+
+        public Uri WebSite => new Uri("https://damianblog.com.com/");
```
Comment (jimbobbennett) [inline src/Firehose.Web/Authors/DamianMehers.cs]: Remove the duplicate .com as this takes you to a scam site

**Comment by DamianMehers** [comment]
Sorry about that - I've fixed it.

**Comment by jfversluis** [comment]
I was looking at (finally) adding your blog. But it seems your website has a certificate error on the https connection?

**Comment by DamianMehers** [comment]
Hi Jim,

No worries I'm a little late replying to you. Indeed the ssl cert is for my
wife's company. Could I use the non https http://damianblog.com instead?

Damian

**Comment by DamianMehers** [comment]
Sorry ... Gerald

**Comment by jfversluis** [comment]
No worries, on both ;)

Sure! We don’t have a strict https policy

**Comment by Cheesebaron** [comment]
@DamianMehers if you don't want to buy a SSL cert, you can always get one free from Let's Encrypt or put your site behind CloudFlare. Just keep in min that Chrome and other browsers are going to complain about your site being insecure. From October Chrome is going to be super aggressive about this.

On the other hand we currently do not have a SSL policy, but we encourage your to get one.

**Pull request merged**